src/main/java/core/basesyntax/model/FruitTransaction.java

```diff
@@ -0,0 +1,48 @@
+package core.basesyntax.model;
+
+public class FruitTransaction {
+    private String fruit;
+    private int quantity;
+
+    public String getFruit() {
+        return fruit;
+    }
+
+    public void setFruit(String fruit) {
+        this.fruit = fruit;
+    }
+
+    public int getQuantity() {
+        return quantity;
+    }
+
+    public void setQuantity(int quantity) {
+        this.quantity = quantity;
+    }
+
+    public enum Operation {

```

Comment:
why this enum is in this class if there is no FruitTransaction field like Operation